core/client/app/templates/components/gh-profile-image.hbs

```diff
@@ -16,5 +16,5 @@
             <span class="sr-only">Upload an image</span>
         </i>
     </span>
-    <input type="file" class="file-uploader js-file-input" name="uploadimage">
+    {{#if fileStorage}}<input type="file" class="file-uploader js-file-input" name="uploadimage">{{/if}}
```

Comment:
okay cool, I had this before (I think) but I wasn't sure I was adding something redundant.